### DISCUSSION TITLE
Time Stamp

### DISCUSSION THREAD

**Discussion by brmeadows2841** [forkineye/ESPixelStick#554] [General]
**Is your feature request related to a problem? Please describe.**

Yes, I use Xlights Connect to upload sequences to an ESP32 Cam. 
The file date and time is wrong on the upload. The time is not the same of the either when the file was created or when it was uploaded.
Im am using FW Version4.0-ci2756477305Build Date Jul 28 2022 - 20:18:03
**Describe the solution you'd like**

File manager in Espixel stick should reflect either the file creation time/date or the upload time/date.

**Additional context**

Would it be possible to add NTP client on boot up to get the correct time?

**Comment by MartinMueller2003** [comment]
Not all show networks have access to the internet (Mine does not and many people prevent this as well). So with NTP not a viable solution and no pins available for an RTC, the decision was made to not use the timestamps for anything useful.

**Comment by brmeadows2841** [comment]
Understood.

But where is the date/time coming from? The correct date is on the files uploaded from Xlights connect but the time is wrong by several hours in the past.

**Reply by MartinMueller2003** [reply]
I try to grab the current time from the connected browser. If the browser disconnects, then I lose my time source. It is cheating but prior to xLights, the only way to download was via the browser so the time would be close. :)
